rest_api/operator_apis/dns-operator-openshift-io-v1.adoc

```diff
@@ -238,12 +238,17 @@ Type::
 |===
 | Property | Type | Description
 
+| `policy`
+| `array (string)`
+| Policy defines the policy to select upstream resolvers. There are 3 values; `Random` is by default and is a policy that implements random upstream selections,`RoundRobin` is a policy that select hosts based on round robin order, and `Sequential` is a policy that select hosts based on sequential order.
```

Comment:
Use : instead of : for first sentence. Use ; instead of commas in the second sentence.

"Random is the default, and is a policy..."

Reply:
re-phrased a bit.